repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,147 @@
+{
+  "type": "object",
+  "properties": {
+    "marathon/libprocess-ip": {
+      "description": "Interface to bind to for communication with the Mesos master.",
+      "type": "string"
+    },
+    "mesos/master": {
+      "description": "The URL of the Mesos master. The format is a comma-delimited list of of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
+      "type": "string"
+    },
+    "marathon/cpus": {
+      "description": "CPU shares to allocate to each Marathon instance.",
+      "type": "number",
+      "minimum": 0.0,
+      "default": 1.0
```

Comment:
I really just mean it wants a non-negative real.  Not sure if this is the right place to guard against ridiculously low resource allocations.